src/context/authStore.test.tsx

```diff
@@ -17,14 +42,22 @@ const testCampaignKey = "test-campaign";
 
 describe("AuthStoreContextProvider", () => {
   beforeEach(() => {
-    mockGetItem.mockReset();
-    mockSetItem.mockReset();
+    mockGetItem.mockReset().mockName("asyncGetItem");
```

Comment:
Does mockName get cleared on a mockReset?

Reply:
Yes, `mockReset` clears everything in the mock function, including implementation, results and the name.